lib/src/googleJavaFormat/java/com/diffplug/spotless/glue/java/GoogleJavaFormatFormatterFunc.java

```diff
@@ -60,7 +65,7 @@ public String apply(@Nonnull String input) throws Exception {
 		// Issue #1679: we used to call ImportOrderer.reorderImports(String) here, but that is deprecated.
 		// Replacing the call with (the correct) reorderImports(String, Style) causes issues for Style.AOSP,
 		// so we force the style to GOOGLE for now (which is what the deprecated method did)
```

Comment:
Remove this comment.

Reply:
580d9c34708bdb863fffa049ccdfe1e7d4baadd4